.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
hmm... how is this `.eslintrc` being used? Is it just for internal linting of this library?